website/versioned_docs/version-5.x/module-types.md

```diff
@@ -0,0 +1,51 @@
+---
+id: module-types
+title: single-spa conceptual module types
+sidebar_label: Conceptual Module Types
+---
+
+# Concept: single-spa modules
```

Comment:
I'm torn on using the term `module` I worry that it's overloaded but I didn't like `types` either for the same reason

Reply:
I agree, module doesn't seem right for our docs. I think it's be best to stay consistent with our current "single-spa applications", or lean into the "microfrontend" or "browser microservice" terminology.

Reply:
Yeah we've already introduced these concepts in other places with other terminology, where we call it "types of microfrontends". The term "single-spa modules" isn't my favorite.

Honestly I wonder if this whole new page would be better off just as additional sections to microfrontends-concept.md? It's only 50 lines of documentation, which isn't a ton.